### ISSUE TITLE
New Message flash should link to a reasonable number of new messages

### ISSUE THREAD

**Issue by grant-olson** [grant-olson/BitMessageForum#12]
I get all excited because I have new messages, but have no idea where to go to read them.  We should link directly to up to 5 of the newest threads.

**Comment by medoix** [comment]
maybe we should have new items displayed on the home page and the new message notification can link back to the home page? Not sure how we can do this as the new messages could be from chans/inbox/lists etc

**Comment by grant-olson** [comment]
Yeah I've been trying to figure out what exactly I want on the back end for an ajax request.

I'm also thinking if there was a page that showed everything under messages, we could just link there.  Then you'd see inbox/chans/etc there with the new status.

**Comment by grant-olson** [comment]
I added information to the ajax request that includes a list of updated folders.  Right now the UX is pretty crappy.  I'm sure you can improve it @medoix .  I wonder if it's possible to use the info to add the new icons back into the menu.  That's just an initial idea.
